homeassistant/components/openrgb/strings.json

```
@@ -0,0 +1,36 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Set up OpenRGB connection",
+        "description": "Enter the host and port of your OpenRGB SDK server",
+        "data": {
+          "name": "[%key:common::config_flow::data::name%]",
+          "host": "[%key:common::config_flow::data::host%]",
+          "port": "[%key:common::config_flow::data::port%]"
+        },
+        "data_description": {
+          "name": "A name for this OpenRGB entry, like the name of the computer running the SDK server.",
+          "host": "The hostname or IP address of the computer running the OpenRGB SDK server.",
+          "port": "The port on which the OpenRGB SDK server is accessible."
```

Some slight improvements
```suggestion
          "host": "Hostname or IP address of the OpenRGB SDK server.",
          "port": "Port number where the OpenRGB SDK server is running."
```

---

https://github.com/home-assistant/core/pull/153373#discussion_r2402686236